### PR TITLE
[Merged by Bors] - feat(MeasureTheory/Borel): add `nullMeasurable_Ixx`

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Measure/MeasureSpace.lean [hunk 1]
```diff
@@ -506,8 +505,7 @@ theorem measure_iInter_eq_iInf' {α ι : Type*} [MeasurableSpace α] {μ : Measu
     · intro h i
       rcases directed_of (· ≤ ·) i i with ⟨j, rij, -⟩
       exact h j i rij
-  have ms : ∀ i, NullMeasurableSet (s i) μ :=
-    fun i ↦ .biInter (to_countable _) fun i _ ↦ h i
+  have ms : ∀ i, NullMeasurableSet (s i) μ := fun i ↦ .biInter (to_countable _) fun i _ ↦ h i
```
Comment (Ruben-VandeVelde) [inline Mathlib/MeasureTheory/Measure/MeasureSpace.lean:508]: While you're here:
```suggestion
  have ms (i) : NullMeasurableSet (s i) μ := .biInter (to_countable _) fun i _ ↦ h i
```

**Review by Ruben-VandeVelde — APPROVED** [review]
lgtm

**Review by Ruben-VandeVelde — APPROVED** [review]
Thanks!

maintainer merge

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**